test.py

```diff
@@ -1,81 +1,53 @@
+import json
 import sys
 from time import time
 from time import sleep
-import argparse
-from argparse import ArgumentParser
-import json
 import numpy as np
 import cv2
+import struct
 
 import depthai
 
 import consts.resource_paths
-from depthai_helpers import utils
-
-def parse_args():
-    epilog_text = '''
-    Displays video streams captured by DepthAI.
-
-    Example usage:
-
-    # Pass thru pipeline config options
-
-    ## USB3 w/onboard cameras board config:
-    python3 test.py -co '{"board_config": {"left_to_right_distance_cm": 7.5}}'
-
-    ## Show the depth stream:
-    python3 test.py -co '{"streams": ["depth_sipp"]}'
-    '''
-    parser = ArgumentParser(epilog=epilog_text,formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument("-co", "--config_overwrite", default=None,
-                        type=str, required=False,
-                        help="JSON-formatted pipeline config object. This will be override defaults used in this script.")
-
-    options = parser.parse_args()
-
-    return options
-
-args = vars(parse_args())
-
-if args['config_overwrite']:
-    args['config_overwrite'] = json.loads(args['config_overwrite'])
-
-print("Using Arguments=",args)
 
 
 cmd_file = consts.resource_paths.device_cmd_fpath
 if len(sys.argv) > 1 and sys.argv[1] == "debug":
     cmd_file = ''
     print('depthai will not load cmd file into device.')
 
+
 labels = []
 with open(consts.resource_paths.blob_labels_fpath) as fp:
     labels = fp.readlines()
     labels = [i.strip() for i in labels]
 
 
-
 print('depthai.__version__ == %s' % depthai.__version__)
 print('depthai.__dev_version__ == %s' % depthai.__dev_version__)
 
 
-
 if not depthai.init_device(cmd_file):
     print("Error initializing device. Try to reset it.")
     exit(1)
 
 
 print('Available streams: ' + str(depthai.get_available_steams()))
 
-# Do not modify the default values in the config Dict below directly. Instead, use the `-co` argument when running this script.
-config = {
-    # Possible streams:
-    # ['left', 'right','previewout', 'metaout', 'depth_sipp']
-    # If "left" is used, it must be in the first position.
-    # To test depth use:
-    # ['metaout', 'previewout', 'depth_sipp']
-    'streams': ['metaout', 'previewout'],
-    'depth':
+
+# Make sure to put 'left' always first. Workaround for an issue to be investigated
+configs = {
+    'streams': [
```

Comment:
Any reason we're showing all of the streams by default? To keep things simple, we just used ['metaout', 'previewout' previously.